doc/source/whatsnew/v1.3.0.rst

```diff
@@ -781,6 +781,7 @@ Groupby/resample/rolling
 - :class:`core.window.ewm.ExponentialMovingWindow` now raises a ``NotImplementedError`` when specifying ``times`` with ``adjust=False`` due to an incorrect calculation (:issue:`40098`)
 - Bug in :meth:`Series.asfreq` and :meth:`DataFrame.asfreq` dropping rows when the index is not sorted (:issue:`39805`)
 - Bug in aggregation functions for :class:`DataFrame` not respecting ``numeric_only`` argument when ``level`` keyword was given (:issue:`40660`)
+- Bug in groupby aggregation when using an object-typed :class:`Index` (issue:`40014`)
```

Comment:
can you be more specific here? as a user i am not sure what i am to do with this statement

Reply:
add a 'for example df.groupby(....)......` or similar, in particular this is only for a user supplied function (e.g. via .apply right)?

Reply:
Adjusted, hope this is more clear?